sdk/communication/communication-chat/samples/javascript/messageOperationSample.js

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * Demonstrates how to use the ChatClient to do thread operations
+ */
+
+import { ChatClient } from "@azure/communication-chat";
+import { AzureCommunicationTokenCredential } from "@azure/communication-common";
+import { CommunicationIdentityClient } from "@azure/communication-identity";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+export const messageOperatioinSample = async () => {
+  const connectionString =
+    process.env["COMMUNICATION_CONNECTION_STRING"] ||
+    "endpoint=https://<resource-name>.communication.azure.com/;<access-key>";
+
+  const identityClient = new CommunicationIdentityClient(connectionString);
+  const testUser = await identityClient.createUser();
+  const userToken = await identityClient.getToken(testUser, ["chat"]);
+
+  //CreateChatClient
+  const chatClient = new ChatClient(
+    connectionString,
+    new AzureCommunicationTokenCredential(userToken.token)
+  );
+  const createChatTtreadResult = await chatClient.createChatThread({ topic: "Hello, World!" });
```

Comment:
typo in the variable name

```suggestion
  const createChatThreadResult = await chatClient.createChatThread({ topic: "Hello, World!" });
```